cmd/lakefs/cmd/expire.go

```diff
@@ -84,7 +85,8 @@ var expireCmd = &cobra.Command{
 				// (not a failure)
 				continue
 			}
-			expiryRows, err := cataloger.QueryExpired(ctx, repo.Name, &policy.Policy)
+			// TODO(ariels): Rewrite this!
```

Comment:
I suggest avoiding TODO comments and open a story instead.. these are action items that aren't tracked.

Reply:
also, why does it require rewriting?